src/python/WMCore/Storage/Backends/GFAL2Impl.py

```diff
@@ -116,17 +112,16 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
         copyCommand = self.copyCommand % copyCommandDict
         result += copyCommand
 
-        if _CheckExitCodeOption:
-            result += """
-            EXIT_STATUS=$?
-            echo "gfal-copy exit status: $EXIT_STATUS"
-            if [[ $EXIT_STATUS != 0 ]]; then
-               echo "ERROR: gfal-copy exited with $EXIT_STATUS"
-               echo "Cleaning up failed file:"
-               %s
-            fi
-            exit $EXIT_STATUS
-            """ % self.createRemoveFileCommand(targetPFN)
+        result += """
```

Comment:
I can understand  your desire to remove this check here for maintaining code simplicity (because `_CheckExitCodeOption` is statically set to `True` at the top of the file and supposedly never changed again). But do you happen to remember/know why was it implemented that way at the first time?